README.md

```diff
@@ -1,28 +1,121 @@
-# SubQuery - Dictionary 
+# 🤔 Introduction
 
-This special SubQuery Project provides a dictionary of data that pre-indexes events on chain to dramatically improve indexing the performance of your own SubQuery Project, sometimes up to 10x faster.
+The Dictionary provides a glossary of data that pre indexes On-Chain events, drastically improving the overall Indexing performance. Sometimes by a factor of 10, which is quite significant.
 
-It scans over the network, and simply records the module and method for every event/extrinsic on each block - please see the standard entities in `schema.graphql`.
+The Dictionary acts as the middleman between the Blockchain and the Indexer. This allows the Indexer to query a block's metadata from the dictionary, allowing one to query blocks for specific events and only return the required blocks.
 
-**If you want to create your SubQuery Dictionary to speed up indexing of your own Substrate chain, fork this project and let us know**
+For example : If one wants to fetch NFT creation events only, the Indexer will ask the dictionary for the corresponding blocks, returning only the blocks where NFT creation took place (5, 9, 32, etc.) instead of all blocks (1 - 100).
 
-# Geting Started
-### 1. Install dependencies
-```shell
-yarn
+## Installation 
+
+#### 1. Clone the Repository
+
+Clone this repository by running this script :
+
+```
+git clone https://github.com/capsule-corp-ternoa/ternoa-subql-dictionary
+```
+
+#### 2. Change Directory
+
+Change the Directory for the desired results : 
+```
+cd ternoa-subql-dictionary
+```
+
+#### 3. Select Testnet version
```

Comment:
Testnet will be soon deprecated.
We should write instead:

#### 3. Select the network environment

Move to the appropriate branch depending of the network environment targeted:

- `v43/mainnet` for Mainnet
- `v43/alphanet` for Alphanet

For example:
```bash
git checkout v43/mainnet
# The indexer and dictionary should be on same network environment.
```

Reply:
Done